server/e2e_test.go

```diff
@@ -503,45 +518,55 @@ func (h *testHarness) AssertP2PBans(bans []testP2PBan) {
 	assert.ElementsMatch(h.t, expectedPairs, actualPairs)
 }
```

Comment:
I don't like unused code, but I forgot this. As mentioned elsewhere, I am currently unable to test p2p bans because I did not find a way to put unique IPs on `net.Pipe()` or real local connections.

Reply:
That seems fine for now. We will figure that out later.